racksdb/web/ui/src/views/DatacentersView.vue

```diff
@@ -11,24 +11,83 @@ import { ref, onMounted } from 'vue'
 import type { Ref } from 'vue'
 import type { Datacenter } from '@/composables/RacksDBAPI'
 import BreadCrumbs from '@/components/BreadCrumbs.vue'
+import L from 'leaflet'
 import ComboBox from '@/components/ComboBox.vue'
+import { useRouter } from 'vue-router'
 
 const http = useHttp()
 const racksDBAPI = useRacksDBAPI(http)
 const datacenters: Ref<Array<Datacenter>> = ref([])
+const map = ref()
+const mapContainer = ref()
+const router = useRouter()
+const longitudes: Array<number> = []
+const latitudes: Array<number> = []
 async function getDatacenters() {
   datacenters.value = await racksDBAPI.datacenters()
+
+  // Average coordinates for correct map adjustment
+  datacenters.value.forEach((datacenter) => {
+    datacenter.location
```

Comment:
I don't see the point of this line of code :thinking:

Reply:
It was a check I forgot to delete!